co2eq/transportation.js

```diff
@@ -12,8 +11,6 @@ import {
   TRANSPORTATION_MODE_FOOT,
 } from '../definitions';
 
-import flightEmissions from './flights';
-
 // ** modelName must not be changed. If changed then old activities will not be re-calculated **
 export const modelName = 'transportation';
```

Comment:
you also need to increment `modelVersion`

Reply:
Done :) I'd suggest updating the README to make the increment requirement more obvious, as I suggested here #364